peps/pep-0773.rst

```diff
@@ -1149,6 +1156,40 @@ substitution of 64-bit binaries. As PyManager runs as a standalone executable,
 this is not a necessary feature for the manager.
 
 
+Test suite and debug symbols
+----------------------------
+
+The existing installer optionally allows installation of the Python standard
+library test suite, and optional installation of debug symbols. Neither of these
+are necessary for most users, but they are convenient for some. Preliminary
+testing shows that omitting the test suite and debug symbols saves about 60% of
+the size of the compressed package (from 46MB to 18MB).
+
+The "default" CPython packages installed by PyManager will therefore not include
+the test suite or the debug symbols. However, there will be a second "company"
+that does include these extras, ``PythonTest`` (as opposed to the default,
```

Comment:
"company" is confusing to me. What does this specifically mean @zooba?

Reply:
It's a term from PEP 514 that refers to the first part of the install identifier - the `PythonCore` in `py install PythonCore\3.14`.

I should _probably_ call it "PEP 514 company" here, though I don't like the PEP number being part of the identity. Every other use of "Company" in the PEP is consistently referring to the same thing, but I'm not real happy with it how it is, so I'll find a way to clarify that doesn't offend me.